Canvas/MainMenu.xib

```diff
@@ -34,6 +35,13 @@
                                     <action selector="orderFrontStandardAboutPanel:" target="-1" id="Exp-CZ-Vem"/>
                                 </connections>
                             </menuItem>
+                            <menuItem isSeparatorItem="YES" id="USI-3R-BOc"/>
+                            <menuItem title="Check for Updates..." id="o6W-LK-QiF">
```

Comment:
I'll probably also add a button in the preferences UI later or you can handle it if you want. (no pressure! 😄)